pyvalidator/is_mobile_number.py

```diff
@@ -271,37 +267,34 @@ class IsMobileNumberOptions(TypedDict):
 mobile_number_patterns['fr-CH'] = mobile_number_patterns['de-CH']
 mobile_number_patterns['it-CH'] = mobile_number_patterns['fr-CH']
 
-__default_options: IsMobileNumberOptions = {
+__default_options = {
```

Comment:
Why is the type definition definition removed?

Reply:
See commit: https://github.com/theteladras/py.validator/pull/9/commits/0fccc93aa4621f24ae9d323a6f9055d1e3bbdf3e

Like we discussed in another script, originally the class of IsMobileNumberOptions is TypedDict, which serves no value here and only serves to cause hint warnings in an IDE. Since just setting it as a _dict_ would be creating a custom type for no reason, it might as well simply be a dict, in which case the IsMobileNumberOptions class serves no purpose at all, hence removing it.

Reply:
It does give a way of leveraging the IDE property hinting, like this:
<img width="559" alt="image" src="https://user-images.githubusercontent.com/31168388/167956086-14195e10-d8b1-40ae-bac0-547248623d04.png">